specifyweb/frontend/js_src/lib/components/PickLists/index.tsx

```diff
@@ -168,7 +168,6 @@ export function PickListComboBox({
           disabled={isDisabled || isReadOnly}
           forwardRef={validationRef}
           name={name}
-          required={parser.required}
```

Comment:
this is needelss because of the `{...getValidationAttributes(parser)}` line above

seems to be here because refactor in
https://github.com/specify/specify7/commit/3ce1509d005ca338e6186816d48a807d80bbc09d#diff-6bae6d2d398090d7c917d80ff60ef98c9c15e765669841b5a2b150069acf8e53R178 didn't simplify this all the way